### DISCUSSION TITLE
How to fuzzy query in tag

### DISCUSSION THREAD

**Discussion by czxin788** [apache/skywalking#9351] [Q&A] [answered]
I found that it only support exact query in tag，how to fuzzy query in tag?
![image](https://user-images.githubusercontent.com/13861904/179643544-0917ec0d-75bc-43d9-a03c-c089996fd37c.png)

**Comment by wu-sheng** [comment, marked as answer]
Then it is not a tag query. The basic concept is, tag is designed for term match.

**Comment by czxin788** [comment]
Thank you.
We wish use http parameters  to   fuzzy query , and obtain these endpoints  which match the  http parameters .
What can we to to realize this goal?

**Reply by wu-sheng** [reply]
I don't know. From my personal view, this is a very bad idea. APM is not a BI system, you should not use in this way.

**Reply by czxin788** [reply]
Thank you.
But I think  it  can be  great help in troubleshooting problems, and it is very necessary.
